### PR TITLE
removes some of the http port cache data as its no longer needed.

### PR DESCRIPTION
**Pull request by matt-raffel-kiva**
Signed-off-by: Matt Raffel <mattr@kiva.org>

### CODE REVIEW

File: src/controller/agent.controller.service.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class AgentControllerService {
         Logger.info(`AgentControllerService.handleRequest(${agentId}, ${route}, ${topic})`, body);
         const agent: any = await this.cache.get(agentId);
```
Comment (jsaur) [inline src/controller/agent.controller.service.ts:25]: Can remove this .gent

Comment (matt-raffel-kiva) [reply src/controller/agent.controller.service.ts:25]: we cannot.  a couple lines later we reference agent to get the adminApiKey

File: src/manager/agent.manager.service.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class AgentManagerService {
         // @tothink move this caching to db
         // adding one second to cache record timeout so that spinDownAgent has time to process before cache deletes the record
         Logger.info(`record cache limit set to: ${(ttl === 0 ? ttl : ttl + 1)}`);
-        await this.cache.set(agentId, { containerId, adminPort, httpPort, adminApiKey, ttl }, {ttl: (ttl === 0 ? ttl : ttl + 1)});
+        await this.cache.set(agentId, { containerId, adminApiKey, ttl }, {ttl: (ttl === 0 ? ttl : ttl + 1)});
```
Comment (jsaur) [inline src/manager/agent.manager.service.ts:76]: We should be able to remove this .set entirely

File: src/manager/agent.manager.service.ts [hunk 3]
```diff
@@ -92,19 +92,17 @@ export class AgentManagerService {
...
     }
 
     /**
      * TODO we should probably respond with something
      */
     public async spinDownAgent(agentId: string) {
-        // TODO: do we need to remove this entry
-        const agent: any = await this.cache.get(agentId);
+        Logger.log('Spinning down agent', agentId);
         await this.cache.del(agentId);
```
Comment (jsaur) [inline src/manager/agent.manager.service.ts:103]: Same with this .del

File: src/manager/agent.manager.service.ts (hunk 2 above)
Comment (matt-raffel-kiva) [reply src/manager/agent.manager.service.ts:76]: We need to cache adminAPIKey for the governance Policy handlers.  Unfortunately we cannot remove setting and removing cache items

File: src/manager/agent.manager.service.ts (hunk 3 above)
Comment (matt-raffel-kiva) [reply src/manager/agent.manager.service.ts:103]: We need to cache adminAPIKey for the governance Policy handlers.  Therefore we have to delete it on stopAgent

File: src/manager/agent.manager.service.ts [hunk 4]
```diff
@@ -186,9 +184,6 @@ export class AgentManagerService {
                 const connectionData = await this.createConnection(agentId, agentData.adminPort, adminApiKey);
```
Comment (jsaur) [inline src/manager/agent.manager.service.ts:184]: I think we can remove the this.cache.get() above sinnce the admin key is passed inn

**Review by jsaur — APPROVED** [review]
Looks good 👍

**Pull request merged**